lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -21,9 +21,50 @@
             <layout.heading local-class='Heading'>
                 <div local-class='Title'>{{t 'registries.drafts.draft.form.new_registration'}}</div>
             </layout.heading>
-            <layout.left local-class='Left'>
-                {{!-- Sidenav: page headings, validations --}}
-            </layout.left>
+            
+            <layout.leftNav as |leftNav|>
+                {{#each draftManager.pageManagers as |pageManager index|}}
+                    <PageLink
```

Comment:
Judging by how this component is invoked, does it make more sense to rename it to `PageLinkManager`?